conda_build/cli/main_debug.py

```diff
@@ -63,6 +63,11 @@ def get_parser() -> ArgumentParser:
         help="Output only the string to the used generated activation script.  Use this for creating envs in scripted "
         "environments.",
     )
+    p.add_argument(
+        "--debug",
```

Comment:
Do we want to add `--verbose` too? Looks like the `debug` subcommand parser extends the `render` subcommand parser, but they are adding `--verbose` separately there.